lib/model/model.go

```diff
@@ -2099,18 +2099,34 @@ func unifySubs(dirs []string, exists func(dir string) bool) []string {
 
 	// Trim each item to itself or its closest known parent
 	for _, sub := range dirs {
-		for sub != "" && sub != ".stfolder" && sub != ".stignore" {
-			if exists(sub) {
-				break
+		if sub == "" || sub == "." || sub == string(filepath.Separator) {
```

Comment:
you are using `string(filepath.Separator)` more than once.
Perhaps perform the type conversion once and store it in a variable.